Apps/WebClient/test/functional/e2e/cypress/integration/timeline/starRating.js

```diff
@@ -0,0 +1,20 @@
+const { AuthMethod } = require("../../support/constants")
+
+describe('Validate Star Rating', () => {
+    beforeEach(() => {
+    })
+
+    it('Cliking the 5 star button should logout', () => {
+        cy.login(Cypress.env('keycloak.username'), Cypress.env('keycloak.password'), AuthMethod.KeyCloak)
```

Comment:
why each test has a login instead of putting it in the beforeEach?

Reply:
fixing